dash/include/dash/Shared.h

```diff
@@ -58,6 +58,23 @@ class Shared {
    * Constructor, allocates shared value at single unit in specified team.
    */
   Shared(
+    /// Unit id of the shared value's owner.
+    team_unit_t   owner = team_unit_t(0),
+    /// Team containing all units accessing the element in shared memory
+    Team     &    team  = dash::Team::All())
+    : Shared(value_type{}, owner, team)
+  {
+    DASH_LOG_TRACE("Shared.Shared(team,owner) >",
+                   "finished delegating constructor");
+  }
+
+  /**
+   * Constructor, allocates shared value at single unit in specified team.
+   * The element is initialized with the given value.
+   */
+  Shared(
+    /// The value to initialize the element with
+    value_type    val,
```

Comment:
Why not pass `val` by reference, i.e., `const valye_type&`?

Reply:
Good point, copied it over from the `set` function,
changed it there as well.